sds_data_manager/lambda_code/SDSCode/pipeline_lambdas/spice_indexer.py

```diff
@@ -320,10 +320,12 @@ def write_data_to_efs(s3_key: str, s3_bucket: str, data_mount_path: Path) -> Pat
         # Download path to the EFS path
         efs_spice_filename_and_path = efs_spice_path / filename
         # Download file from S3 to the EFS path
+        logger.info(f"Downloading {s3_key} to {efs_spice_filename_and_path}")
         s3_client.download_file(s3_bucket, s3_key, efs_spice_filename_and_path)
-        logger.info(f"{s3_key} file downloaded successfully")
+        logger.info("Download Successfull")
     except Exception as e:
         logger.error(f"Error downloading file: {e!s}")
```

Comment:
Remove the logger.error if you are raising a ValueError here with the same info below?